utils/build-presets.ini

```diff
@@ -1701,6 +1701,26 @@ assertions
 swiftsyntax
 swiftsyntax-verify-generated-files
 
+#===------------------------------------------------------------------------===#
+# Test Swift Format
+#===------------------------------------------------------------------------===#
+
+[preset: buildbot_swiftformat_macos]
+mixin-preset=mixin_swiftpm_package_macos_platform
+release
+assertions
+swiftsyntax
+skip-test-swiftsyntax
```

Comment:
IIRC there is no need to specify the build of SwiftSyntax here. The build system will automatically pick up the dependency on SwiftSyntax from the `.xcworkspace`.